sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -191,7 +194,9 @@ def set_split_points_unclaimed_callback(self, callback):
 
 
 class GroupedShuffleRangeTracker(iobase.RangeTracker):
-  """A 'RangeTracker' for positions used by'GroupedShuffleReader'.
+  """For internal use only; no backwards-compatibility guarantees.
```

Comment:
The thinking here was that if this is not public, it might be good to have to warning about not being backwards compatible as the summary. SDK developers can read rest of the comment to learn more details. Everybody else can stop at this line.